serverless/cmd/feeder/github/main.go

```diff
@@ -201,6 +203,18 @@ func feedOnce(ctx context.Context, opts *options, forkRepo *git.Repository, ghCl
 	return createPR(ctx, opts, ghCli, fmt.Sprintf("Witness @ %d", wCp.Size), opts.feederRepo.owner+":"+branchName, "master")
 }
 
+// cpID returns a stable identifier for a given checkpoint and set of known signatures.
+// This is used as a branch name, and for generating a file name for the witness PR.
+func wcpID(n note.Note) string {
+	bits := []string{n.Text}
```

Comment:
Maybe parts or components instead of `bits`? I genuinely thought you were going to do some bitwise magic here to make this work.

Reply:
done